packages/js/product-editor/src/components/feedback-bar/style.scss

```diff
@@ -40,3 +42,52 @@
 		}
 	}
 }
+.components-modal__content {
+	> p {
+		text-transform: uppercase;
+		color: $gray-900;
+	}
+	.woocommerce-customer-effort-score__selection {
+		.components-radio-control__option {
+			margin-right: 0;
+			label {
+				color: $gray-900;
+				padding: 1em 0;
+				width: 8.8em;
+				height: 80px;
+				&::before {
+					margin: 8px 0;
+				}
+				&:hover {
+					box-shadow: 0 0 0 1.5px $gutenberg-blue;
+					border-radius: 2px;
+					background-color: #fff;
+				}
+			}
+		}
+	}
+	.components-base-control {
+		margin-top: 10px;
```

Comment:
We don't have a variable for 10px but if it was possible to use either 8px or 12px, I would suggest using `$gap-smaller` or `$gap-small`